project_sla/report/report_sla.py

```diff
@@ -0,0 +1,98 @@
+from openerp import tools
+from openerp.osv import fields, orm
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+SLA_STATES = [('5', 'Failed'), ('4', 'Will Fail'), ('3', 'Warning'),
+              ('2', 'Watching'), ('1', 'Achieved')]
+
+
+class report_sla(orm.Model):
```

Comment:
Not a requirement: many official reports use views a + transient models.
But I can see the flexibility on doing it as a regular model, so I'm OK with it.